plugins/foreman_chef/0.1/index.md

```diff
@@ -0,0 +1,474 @@
+---
+layout: plugin
+title: Foreman Chef 0.1 Manual
+version: 0.1
+---
+
+# 1. {{ page.title }}
+
+This plugin enables Foreman to receive reports and attributes from chef clients
+and use these data for usual Foreman features like displaying status of your
+infrastructure on dashboard, monitoring based on convergence report, 
+boostraping chef-client during host provisioning etc. This plugin also make
+active communication with chef server possible so we can delete nodes and
+clients upon host deletion in Foreman.
+
+All communication is through Foreman Proxy which is running a [separate plugin](https://github.com/theforeman/smart_proxy_chef) and has
+direct access to the Chef server and Foreman instance. Chef clients must be
+able to connect to Foreman Proxy while the opposite direction is not required. The upload is triggered at the end of each chef client run, which is managed by [third plugin](https://github.com/theforeman/chef-handler-foreman) in form of chef handler, also provided by Foreman.
+
+![Infrastructure overview](infrastructure_schema.svg)
```

Comment:
You'll need to put the full page in here, as I think we have a `<base>` tag that makes it try and load from root.